samples/cloudwatch/reference.md

```diff
@@ -0,0 +1,2926 @@
+# CloudWatch SAS Viya Metrics
+
+## By Dimensions
+This table lists the metrics associated with each set of dimensions.
+<details>
+  <summary>Click to expand</summary>
+<hr/>
+<details>
+  <summary>ClusterName,cas_node,cas_node_type,cas_server,job,namespace,pod (8)</summary>
+
+| Metric | Source |
+| ------ | ------ |
+| cas_node_fifteen_minute_cpu_load_avg | sas-cas |
+| cas_node_five_minute_cpu_load_avg | sas-cas |
+| cas_node_free_files | sas-cas |
+| cas_node_inodes_free | sas-cas |
+| cas_node_inodes_used | sas-cas |
+| cas_node_max_open_files | sas-cas |
+| cas_node_one_minute_cpu_load_avg | sas-cas |
+| cas_node_open_files | sas-cas |
+
+</details>
+<hr/>
+<details>
+  <summary>ClusterName,cas_node,cas_node_type,cas_server,connected,job,namespace,pod,uuid (1)</summary>
+
```

Comment:
It would be nice to use different formatting and/or add text (maybe _**Dimensions:**_) to make the headers standout (or appear different) than the "Click to expand" text.  Once you expand, you see additional expandable sections but what the headers are confusion unless you know what they are.

Reply:
Changed 'Click to expand' to the name of the field - 'Dimensions', 'Sources', ...